eo-maven-plugin/src/main/java/org/eolang/maven/footprint/FtCached.java

```diff
@@ -103,6 +104,11 @@ public void save(final String program, final String ext, final Scalar<String> co
         this.origin.save(program, ext, () -> text);
     }
 
+    @Override
+    public List<Path> list(final String ext) throws IOException {
```

Comment:
@volodya-lombrozo Need to cover it with tests. It's interesting what should be returned for cached version.

Reply:
@mximp I've left behaviour from a delegate object. In the future we will able to change it (if somebody needs it).
Also I've added test.